finetune.py

```diff
@@ -322,7 +322,7 @@ def finetune(model, train_loader, train_hiddens, args, device, val_loader=None,
         train_dataloader = dataloader
         val_dataloader = None
     # create original model
-    orig_model = get_model(args.base_model, None, args.dtype, args.model_seqlen, args.device_map)
+    orig_model = get_model(args.base_model, None, args.dtype, args.device_map)
```

Comment:
rationale: do not attach seqlen as a monkey-patched field of the model; instead, use args.model_seqlen directly.